core/src/test/java/org/apache/calcite/test/DiffRepository.java

```diff
@@ -756,12 +863,16 @@ public static DiffRepository lookup(
    * @param clazz     Test case class
    * @param baseRepository Base repository
    * @param filter    Filters each string returned by the repository
+   * @param indent    Indent of the XML file (usually 2)
+   * @param expectedOutOfOrderTests List of {@code <TestCase>} elements that
```

Comment:
expectedOutOfOrderTests is always 0. Would you please remove the overload to avoid accidental complexity?